### DISCUSSION TITLE
Packaging and compiling under the Unity URP pipeline is very slow, there are hundreds of thousands of shader variants, which can take several hours to compile.

### DISCUSSION THREAD

**Discussion by hcshen** [atteneder/glTFast#638] [General]
Packaging and compiling under the Unity URP pipeline is very slow, there are hundreds of thousands of shader variants, which can take several hours to compile.

**Comment by WeeJeePee** [comment]
same here this appears to still be a problem

**Comment by atteneder** [comment]
If you can, please only add shader variants to you build that you actually need.

Granted, the tooling around this can definitely be improved.

For cases where a limited variant set is not an option, making a shader with reduced keywords might be the solution. See #703 .
